### ISSUE TITLE
Crash: C-style comment start token without matching end token

### ISSUE THREAD

**Issue by caitp** [google/traceur-compiler#1809]
As mentioned here, https://github.com/google/traceur-compiler/pull/1808#issuecomment-78156715

The following will trigger a hang/crash in the repl, and also crashes in node.

```js
ar fn = (a) /*
  => a
 
fn(1);
```

(or, [as seen on repl](https://google.github.io/traceur-compiler/demo/repl.html#var%20fn%20%3D%20(a)%20%2F*%0A%20%20%3D%3E%20a%0A%20%0Afn(1)%3B))

**Comment by johnjbarton** [comment]
Infinite loop
```js
function peekTokenNoLineTerminator() {
    var t = peekToken();
    var start = lastToken.location.end.offset;
    var end = t.location.start.offset;
    for (var i = start; i < end; i++) {
      var code = input.charCodeAt(i);
      if (isLineTerminator(code))
        return null;
      if (code === 47) {
        code = input.charCodeAt(++i);
        if (code === 47)
          return null;
        i = input.indexOf('*/', i) + 2; // << if there is no */ we set i = 2
      }
    }
    return t;
  }
```

**Comment by arv** [comment]
This doesn't happen with #1808. I'll add a test to that.

**Comment by caitp** [comment]
Think this was fixed in 04a4481cad4bc944456fbcd1f3f1f77a5903acfa =)
